components/button-group/button-group.js

```diff
@@ -0,0 +1,162 @@
+/* eslint-disable sort-class-members/sort-class-members */
+import '../colors/colors.js';
+import '../dropdown/dropdown-menu.js';
+import '../button/button.js';
+import '../button/button-subtle.js';
+import '../dropdown/dropdown.js';
+import '../dropdown/dropdown-button.js';
+import '../dropdown/dropdown-button-subtle.js';
+import '../dropdown/dropdown-more.js';
+import '../menu/menu.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { DynamicResizeMixin } from '../../mixins/dynamic-resize-mixin.js';
+import { LocalizeCoreElement } from '../../lang/localize-core-element.js';
+import { offscreenStyles } from '../offscreen/offscreen.js';
+import { RtlMixin } from '../../mixins/rtl-mixin.js';
+
+/**
+ *
+ * A button group component that can be used to display a set of buttons
+ *
+ * @slot - Buttons, dropdown buttons, links or other items to be added to the container
+ */
+class ButtonGroup extends RtlMixin(LocalizeCoreElement(DynamicResizeMixin(LitElement))) {
+
+	static get properties() {
+		return {
+			/**
+			 * Setting this property to true will change the look of the dopdown button to a subtle button for subtle button groups
+			 */
+			subtle: {
+				type: Boolean,
+				attribute: 'subtle',
+			},
+
+		};
+	}
+
+	static get styles() {
+		return [offscreenStyles,
+			css`
+			:host {
+				display: block;
+
+			}
+			
+			:host([subtle]) {
+				--d2l-button-icon-fill-color: var(--d2l-color-celestine);
+				--d2l-button-icon-fill-color-hover: var(--d2l-color-celestine-minus-1);
+			}
+
+			:host .d2l-button-group-container {
```

Comment:
No need to prefix any of these with `:host` -- the way shadow DOM works all these styles will be scoped to the host anyway.